src/main/resources/banner.txt

```diff
@@ -0,0 +1,31 @@
+tasks.withType(Checkstyle) {
+	configFile file('./codequality/checkstyle.xml')
```

Comment:
Remove this file content and add these in their right places

Reply:
fixed